Tests/anvil-csharp-tests.asmdef

```diff
@@ -1,7 +1,9 @@
 {
     "name": "anvil-csharp-tests",
     "rootNamespace": "Anvil.CSharp.Tests",
-    "references": [],
+    "references": [
+        "anvil-csharp-core"
+    ],
```

Comment:
@jkeon You were right - a Unity assembly definition will auto-reference DLL's, but not other assembly definitions. I inadvertently tested this unit test project by writing a Logging test... which I was only able to access because it's already compiled into a DLL! This reference makes the rest of Anvil accessible (in Unity).

Reply:
Ok good to know!